ROLES.md

```diff
@@ -232,31 +232,44 @@ approver in an OWNERS file:
 
 ## Lead
 
-Working group leads, or just ‘leads’, are approvers of an entire area that have
-demonstrated good judgement and responsibility. Leads accept design proposals
-and approve design decisions for their area of ownership.
+Working group leads, or just ‘leads’, coordinate working group processes and
+decision-making. Leads are responsible for selecting appropriate processes and
```

Comment:
replace the trailing ` and` with `,`

Reply:
I think this was intended to be:

> Leads are responsible for:
> * Selecting appropriate processes
> * Tracking and reporting on the working group's results to the TOC

Not:

> Leads are responsible for:
> * Selecting appropriate processes
> * Tracking
> * Reporting on the working group's results to the TOC.